lib/models/BucketInfo.ts

```diff
@@ -59,7 +60,7 @@ export default class BucketInfo {
     _objectLockEnabled?: boolean;
     _objectLockConfiguration?: any;
     _notificationConfiguration?: any;
-    _tags?: { key: string; value: string }[] | null;
+    _tags?: Array<BucketTag> | [];
```

Comment:
Do we need the `| []`? I believe an empty array should still have the bucket tag type so we wouldn't need this or empty array thing. Could be wrong though.